packages/admin/blocks-admin/jest.config.ts

```diff
@@ -144,7 +144,9 @@ export default {
     testEnvironment: "jest-environment-jsdom",
 
     // Options that will be passed to the testEnvironment
-    // testEnvironmentOptions: {},
+    testEnvironmentOptions: {
+        customExportConditions: ["browser", "node", "node-addons"],
+    },
```

Comment:
Why is this necessary?

Reply:
Without this change the tests would fail: https://github.com/vivid-planet/comet/actions/runs/12197580622/job/34027526177?pr=2846#step:10:341

@manuelblum helped me fixing it with this change, may he can explain better why this is necessary.